server/src/testFixtures/java/io/crate/testing/SymbolAssert.java

```diff
@@ -121,14 +121,14 @@ public SymbolAssert isReference(String expectedName) {
         return this;
     }
 
-    public SymbolAssert isReference(ColumnIdent expectedColumn,
+    public SymbolAssert isReference(String expectedName,
                                     RelationName expectedRelName,
                                     DataType<?> expectedType) {
         isNotNull();
         isInstanceOf(Reference.class);
-        assertThat(((Reference) actual).column())
-            .as("column")
-            .isEqualTo(expectedColumn);
+        assertThat(((Reference) actual).column().sqlFqn())
+            .as("sqlFqn")
+            .isEqualTo(expectedName);
```

Comment:
I'd keep this as ColumnIdent. Using a string has ambiguity in regards to the format (dotted, vs. [] notation)